### PR TITLE
Fix for quotations

### PR DESCRIPTION
**Pull request by PiotrKozlowski**
- Insert paragraph when there is only `title` inside `quotation`.
- Change second `title` to `paragraph` - ex. when user hit `enter` inside `title`.
- Add test for first change and update test for duplicate titles.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "editor.rulers": [80,120]
+}
```
Comment (aiwenar) [inline .vscode/settings.json]: Remove this file.

File: src/plugins/quotation/schema.js [hunk 2]
```diff
@@ -2,21 +2,36 @@
...
+            case 'title':
+                change.insertNodeByKey(node.key, 1, {
+                    object: 'block',
+                    type: 'paragraph',
+                    nodes: [Text.create('')],
+                })
+                break
+            }
+            
             return
```
Comment (aiwenar) [inline src/plugins/quotation/schema.js]: This will ignore errors not matching these criteria.
